components/appgateway/main.tf

```diff
@@ -0,0 +1,33 @@
+module "logworkspace" {
+  source      = "git::https://github.com/hmcts/terraform-module-log-analytics-workspace-id.git?ref=master"
+  environment = var.env
+
```

Comment:
Remove blank line

Reply:
removed - though does not affect tf format or lint checks